exporters/otlp/src/otlp_grpc_client.cc

```diff
@@ -392,7 +392,11 @@ std::shared_ptr<grpc::Channel> OtlpGrpcClient::MakeChannel(const OtlpGrpcClientO
   }
 #endif  // ENABLE_OTLP_RETRY_PREVIEW
 
-  if (options.use_ssl_credentials)
+  if (options.credentials)
+  {
+    channel = grpc::CreateCustomChannel(grpc_target, options.credentials, grpc_arguments);
+  }
+  else if (options.use_ssl_credentials)
```

Comment:
Could a warning be emit here if both `credentials` and `use_ssl_credentials` are configured in the option, to notify that `credential` is preferred.

Reply:
Done